META.json

```diff
@@ -0,0 +1,46 @@
+{
+   "name": "ApacheAGE",
```

Comment:
Can there be a space between Apache and AGE?

Reply:
Not Possible, I tried that PGXN does not support.